Create class car

maver1cus opened the pull request.

maver1cus committed:
реализует класс Car (синтаксист ES6)

maver1cus committed:
реализует класс Car (синтаксист ES5)

kicumkicum commented on index.js:
```diff
@@ -0,0 +1,16 @@
+carEs5.ride();
+carEs5.stop();
+carEs5.turnRight();
+carEs5.turnLeft();
```
Нужно добавить перенос строки в конце файла, где его нет. Чтобы не следить за этим и не добавлять руками каждый раз, можно добавить в проект `.editorconfig`. Дефолтный конфиг поддерживает эту настройку https://editorconfig.org .

maver1cus replied:
fixed

kicumkicum commented on src/car.js5.js:
```diff
@@ -0,0 +1,17 @@
+export default function Car() {}
```
Файлы нужно было назвать `car.es5.js` и `car.es6.js`

maver1cus replied:
fixed

kicumkicum commented on src/car.js6.js:
```diff
@@ -0,0 +1,19 @@
+
+
+    turnRight(){
+        console.log(`ride`);
```
В console.log нужно написать `turnRight`

maver1cus replied:
fixed

kicumkicum commented on src/car.js6.js:
```diff
@@ -0,0 +1,19 @@
+        console.log(`ride`);
+    }
+
+
```
Где-то два переноса, а где-то один. Нужны выбрать единый стиль. Позже ты подключишь линтер и он будет делать это автоматически. Но пока за стилем написания нужно следить самому

maver1cus replied:
fixed

kicumkicum commented on src/car.js6.js:
```diff
@@ -0,0 +1,19 @@
+    }
+
+
+    turnRight(){
```
Стоит добавить пробел между `){`, по аналогии с `function`

maver1cus replied:
fixed

kicumkicum reviewed (commented):
Все очень хорошо 👍🏽 

- [x] Нужно исправить несколько комментов по стилю. Когда будешь фиксить коммент, то напиши сообщение `done`, `fixed`, `сделано` или любое другое. Так я узнаю о твоем прогрессе, а тебе будет проще ориентироваться.

- [x] Еще стоит фиксануть заголовок ПРа. Удалить `feature` и написать `create` с большой буквы

maver1cus committed:
fix:
  - добавляет .editorconfig
  - добавляет .gitignore
  - переименовывает файлы car.js5.js и car.js6.js в car.es5.js и car.es5.js
  - исправляет метод turnRight класса Car в car.es6.js
  - исправляет отступы и пробелы в car.es6.js

kicumkicum reviewed (approved):
👍🏽
